src/Documentation/glossary.js

```diff
@@ -8,16 +8,25 @@ export default {
       name: 'Workspace',
       match: ['workspace'],
       desc: `
-By "workspace" we refer to the directory containing all your project files. For
-example raw datasets, source code, ML models, etc. A workspace becomes a DVC
-project when [\`dvc init\`](/doc/commands-reference/init) is run, and
-[DVC-files](/doc/user-guide/dvc-file-format) are created in it. It\s typically
-also a Git repository.
+Directory containing all your project files. For example raw datasets, source
+code, ML models, etc. A workspace becomes a **DVC project** when
+[\`dvc init\`](/doc/commands-reference/init) is run, and
+[DVC-files](/doc/user-guide/dvc-file-format) are created in it.
```

Comment:
probably better to point to https://dvc.org/doc/user-guide/dvc-files-and-directories

Reply:
Hm. This refers to DVC-files e.g. `stage.dvc` a.k.a. stage files. Not DVC files (and directories). Are you sure?

Reply:
p.s. Added " or stage files" in 9206bdf to clarify, for now.

Reply:
👍